src/sentry/tasks/derive_code_mappings.py

```diff
@@ -102,12 +102,6 @@ def derive_code_mappings(
         logger.info("Event should not be processed.", extra=extra)
         return
 
-    # php automatic code mappings currently in LA
-    if data["platform"].startswith("php") and not features.has(
-        "organizations:derive-code-mappings-php", org
-    ):
-        return
-
```

Comment:
ideally this change would be in a separate PR, and the order of landing would be:
1. PR removing these lines 
2. Options automator PR
3. GetSentry PR
4. the rest of this PR cleaning up the flag. 

That being said, removing value from options-automator does not automatically remove it, so an OK way would be:
1. Options automator PR
2. GetSentry PR
3. This PR